UI/cmake/os-linux.cmake

```diff
@@ -1,5 +1,8 @@
 target_sources(obs-studio PRIVATE platform-x11.cpp)
-target_compile_definitions(obs-studio PRIVATE USE_XDG OBS_INSTALL_PREFIX="${OBS_INSTALL_PREFIX}")
+target_compile_definitions(
+  obs-studio
+  PRIVATE USE_XDG OBS_INSTALL_PREFIX="${OBS_INSTALL_PREFIX}" $<$<BOOL:ENABLE_PORTABLE_CONFIG>:ENABLE_PORTABLE_CONFIG>
```

Comment:
This line is wrong, the `$<BOOL>` expression requires variable notation here - will push a fix in a bit.